apps/trading/pages/client-router.tsx

```diff
@@ -35,6 +35,14 @@ const LazyDeposit = dynamic(() => import('../client-pages/deposit'), {
   ssr: false,
 });
 
+const LazyWithdraw = dynamic(() => import('../client-pages/withdraw'), {
```

Comment:
I'd like to move away from using next/dynamic and even React.lazy in favor of just having a single js bundle. This makes things easier with ipfs and the benefits we gain from code splitting are minimal due to the nature of this being more app-like

Reply:
I've changed these to just be imported in a regular fashion. I'll make an additional PR to convert the rest